README.md

```diff
@@ -34,3 +43,21 @@ The server will store some statistics data every 12 hours for analysing the traf
 - The number of push notifications sent to Android devices
 
 There is also an endpoint `/get_statistics_data` to get the data above.
+
+
+## Potential issues
+
+If you get an issue during the `sudo apt update` about the certificate chain being invalid,
+try to create a file at `/etc/apt/apt.conf.d/99deboxenio-cert` with content:
+`Acquire::https::deb.oxen.io::Verify-Peer "false";`
```

Comment:
This is a dangerous suggestion and isn't necessary: instead the solution here is to install system updates first (which will fix the broken `ca-certificates` package).

Reply:
:+1: